src/main/kotlin/io/github/detekt/compiler/plugin/DetektCommandLineProcessor.kt

```diff
@@ -75,24 +75,24 @@ class DetektCommandLineProcessor : CommandLineProcessor {
 
     override fun processOption(option: AbstractCliOption, value: String, configuration: CompilerConfiguration) {
         when (option.optionName) {
-            Options.baseline -> configuration.put(Keys.BASELINE, value)
-            Options.config -> configuration.put(Keys.CONFIG, value)
+            Options.baseline -> configuration.put(Keys.BASELINE, Paths.get(value))
+            Options.config -> configuration.put(Keys.CONFIG, value.split(",;").map { Paths.get(it) })
             Options.debug -> configuration.put(Keys.DEBUG, value.toBoolean())
             Options.isEnabled -> configuration.put(Keys.IS_ENABLED, value.toBoolean())
-            Options.useDefaultConfig -> configuration.put(Keys.USE_DEFAULT_CONFIG, value)
+            Options.useDefaultConfig -> configuration.put(Keys.USE_DEFAULT_CONFIG, value.toBoolean())
```

Comment:
I thought that this was not configurable in this repo.

Reply:
It's still configurable, but set to `true` by default, where in the existing detekt plugin it's `false` by default.